components/server/src/workspace/workspace-starter.ts

```diff
@@ -2041,6 +2044,17 @@ export class WorkspaceStarter {
         const auth = new BuildRegistryAuth();
         auth.setTotal(allowAll);
         req.setAuth(auth);
+
+        // if the image resolution is for an organization, we also include the organization's set up env vars
+        if (organizationId) {
+            await this.auth.checkPermissionOnOrganization(user.id, "read_env_var", organizationId);
+            const orgEnvVars = await this.orgDB.getOrgEnvironmentVariables(organizationId);
```

Comment:
I feel this should be rather `EnvVarService.listOrgEnvVars`  + `EnvVarService.getOrgEnvVarWithValue` (or sth) instead, even though the later needs to be added.

@filiptronicek WDYT?

Reply:
@geropl klingt gut, I'm going to implement `listOrgEnvVarsWithValues` on the `EnvVarService` to combine these to and make it easier to read.

Reply:
Did it in https://github.com/gitpod-io/gitpod/pull/20560/commits/719452234e8c19913887ebac4d07e42841631eb7